src/deadline/client/api/_attachment.py

```diff
@@ -0,0 +1,82 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+import os
+import boto3
+import json
+
+from contextlib import ExitStack
+from typing import Optional, List, Dict
+
+from deadline.job_attachments.asset_manifests.base_manifest import BaseAssetManifest
+from deadline.job_attachments.asset_manifests.decode import decode_manifest
+from deadline.job_attachments.download import download_files_from_manifests
+from deadline.job_attachments.models import JobAttachmentS3Settings, PathMappingRule
+from deadline.job_attachments.progress_tracker import DownloadSummaryStatistics
+from deadline.client.exceptions import NonValidInputError
+
+
+def attachment_download(
+    manifests: List[str],
+    s3_root_path: str,
+    boto3_session: boto3.Session,
+    path_mapping_rules: Optional[str] = None,
+):
+    """
+    ALPHA API - This API is still evolving but will be made public in the near future.
+
+    API to download job attachments based on given list of manifests.
+    If path mapping rules file is given, map to corresponding destinations.
+
+    Args:
+        manifests: File Path to the manifest file for upload.
+        s3_root_path: S3 root path including bucket name and root prefix.
```

Comment:
sorry `s3_cas_path`

Reply:
This is the s3 bucket name + root prefix though. In this case what about `s3_bucket_root_path` ? The name may forego the S3 bucket name.

Reply:
Synced offline, we'll run it by others to get more data what would be a user friendly name. Until then, will keep it as is.

Reply:
We could also consider just using S3 URI. It has the information we want in it, just need to trim off the "which URI is it identifier" part. This is copy/pastable from the S3 console